server/server.py

```diff
@@ -186,42 +259,31 @@ def uploadSticker():
     else:
         return json.dumps({'status': '405', 'error': 'HTTP Method not allowed.'}), 405
 
-# @app.route('/addEmail', methods=['PATCH'])
-# def addEmail():
-#     # Check token if required
-#     if os.getenv('STICKER_MAP_REQUIRE_LOGIN') == "True":
-#         if not checkToken(request.cookies.get('token')):
-#             return json.dumps({'status': '403', 'error': 'Not authenticated or cookies disabled.'}), 405
-#     print(f"{request.form["email"]}, {request.form["token"]}")
-#     if request.form['email'] != '':
-#         if request.form['token'] != '':
-#             # Check if the token is in the database
-#             with Session(engine) as session:
-#                 stmt = select(Sticker).where(Sticker.adderemail.is_(request.form['token']))
-#                 sticker = session.query(Sticker).from_statement(stmt).one_or_none()
-#                 print(sticker)
-#                 if sticker is not None:
-#                     # Change email in database
-#                     sticker.adderemail = request.form["email"]
-#                     session.commit()
+@app.route('/addEmail', methods=['PATCH'])
+def addEmail():
+    if request.form['email'] != '':
+        if request.form['token'] != '':
+            # Check if the token is in the database
+            with Session(engine) as session:
+                stmt = select(Sticker).where(Sticker.adderemail.is_(request.form['token']))
+                sticker = session.query(Sticker).from_statement(stmt).one_or_none()
+                if sticker is not None:
+                    # Change email in database
+                    sticker.adderemail = request.form["email"]
+                    session.commit()
                     
-#                     return json.dumps({'status': '200', 'error': 'Email updated in database.'}), 200
-#                 else:
-#                     return json.dumps({'status': '400', 'error': 'Token not valid.'}), 400
-#         else:
-#             return json.dumps({'status': '400', 'error': 'Token not defined.'}), 400
-#     else:
-#         return json.dumps({'status': '400', 'error': 'Email not defined'}), 400
+                    return json.dumps({'status': '200', 'error': 'Email updated in database.'}), 200
+                else:
+                    return json.dumps({'status': '400', 'error': 'Token not valid.'}), 400
+        else:
+            return json.dumps({'status': '400', 'error': 'Token not defined.'}), 400
+    else:
+        return json.dumps({'status': '400', 'error': 'Email not defined'}), 400
 
 
 @app.route('/getStickers', methods=['GET'])
 def getStickers():
-    # Check token if required
-    if os.getenv('STICKER_MAP_REQUIRE_LOGIN') == "True":
-        if not checkToken(request.cookies.get('token')):
-            return json.dumps({'status': '403', 'error': 'Not authenticated or cookies disabled.'}), 405
```

Comment:
You removed this check, but shouldn't a `@require_login` or something be added instead?